shared/test-esm-exports/package.json

```diff
@@ -9,9 +9,9 @@
   "scripts": {
     "build": "esbuild ./hls.mjs ./mux-embed.mjs --bundle --outdir=dist --out-extension:.js=.js --format=esm"
   },
-  "dependencies": {
-    "hls.js": "1.2.3",
-    "mux-embed": "^4.7.0"
+  "peerDependencies": {
```

Comment:
NOTE: This change was to make version maintenance easier for `test-esm-exports`. Since this package is, by design, intended to rely on the versions of hls.js & mux-embed used in the "real" packages, declaring these versions as `peerDependencies` of `"*"` should more accurately describe their usage here.